llvm/test/tools/llvm-exegesis/RISCV/serialize-obj-file.test

```diff
@@ -0,0 +1,33 @@
+# RUN: llvm-exegesis -mtriple=riscv64 -mcpu=sifive-p470 --opcode-name=SH3ADD --benchmark-phase=assemble-measured-code --mode=latency --benchmarks-file=%t.yaml
+# RUN: FileCheck --input-file=%t.yaml %s --check-prefixes=CHECK,SERIALIZE
```

Comment:
Can you add some more comments in the test on what exactly you're testing for in different sections.

Reply:
Done